src/Subscriber/Mock.php

```diff
@@ -60,6 +60,12 @@ public function onBefore(BeforeEvent $event)
             }
         }
 
+        $save_to = $event->getRequest()->getConfig()->get('save_to');
+
+        if ($save_to) {
+            file_put_contents($save_to, $item->getBody());
```

Comment:
Yeah, sorry about that, totally forgot. here's the updated PR with all the 3 use cases